build.gradle

```diff
@@ -0,0 +1,77 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+sourceCompatibility = 17
+targetCompatibility = 17
```

Comment:
No, we need to target 16 at the minium. Ideally still 8 as upstream does.

Reply:
This uses 17 because `Modifier.SEALED` (https://github.com/liachmodded/javapoet/commit/087692f31acefb3e37c6b073d20770589c646791#diff-272d767148bf7f1f43dc53a073904304501c478b8c7cb8a02750ec1389d49d0bR908) is otherwise unavailable. Might need better way to obtain that.

Reply:
I will use `Modifier.valueOf` and catch IAE for that.